documentation/tutorials/Demo_02_Uncertainty_Analysis/uncertainty.qmd

```diff
@@ -0,0 +1,597 @@
+- **Sensitivity Analysis**: Repeats numerous model runs to assess how changes in model parameters will affect model outputs. Allows us to identify which parameters the model is most sensitive to.
+- **Uncertainty Analysis**: Combines information about model sensitivity with information about parameter uncertainty to determine the contribution of each model parameter to the uncertainty in model outputs. Allow us to identify which parameters are driving model uncertainty.
+
+This demo shows how to run an uncertainty analysis workflow in PEcAn using an R-based Quarto notebook. It covers loading settings, configuring models, running simulations, and performing ensemble and sensitivity analyses to assess uncertainty and parameter importance. This programmatic approach complements the web-based PEcAn interface.
+
+**Context & modeling scenario:**
+
+We simulate plant and ecosystem carbon balance (Net Primary Productivity and Net Ecosystem Exchange) at the AmeriFlux Niwot Ridge Forest site ([US‑NR1](https://ameriflux.lbl.gov/sites/siteinfo/US-NR1)) during the year 2004. We use SIPNET parameterized as a temperate conifer PFT and driven by AmeriFlux meteorology following the analysis in [Moore et al. (2007)](https://doi.org/10.1016/j.agrformet.2008.04.013). This notebook also provides a compact template that can be extended to more years, locations, and PFTs.
+
+**What this notebook does:**
+
+1. Configure a PEcAn workflow by loading and validating a `pecan.xml` settings file.
+2. Run a set of ecosystem model simulations by writing model configuration files and then running SIPNET.
+3. Quantify uncertainty using ensemble analysis, sensitivity analyses, and variance decomposition.
+4. Visualize results to identify important parameters and how they influence model variance.
+5. Change configuration settings and re-run the workflow.
+
+## Prerequisites
+
+To run this notebook, you will need to install PEcAn and its dependencies, as well as download the SIPNET model binary.
+
+### PEcAn packages and dependencies.
+
+```
+# Enable repository from pecanproject
+options(repos = c(
+  pecanproject = 'https://pecanproject.r-universe.dev',
+  CRAN = 'https://cloud.r-project.org'))
+install.packages('PEcAn.all')
+```
+
+A valid `pecan.xml` configuration file. Start with the example at `pecan/documentation/tutorials/Demo_02_Uncertainty_Analysis/pecan.xml`.
+
+### SIPNET v1.3.0
+
+If you haven't already installed the SIPNET model, you can do so by running the following code. This will download the SIPNET binary to `demo_outdir/sipnet` and make it executable.
+
+> Note: The `demo_outdir` directory will be created in the root of your PEcAn installation (i.e., at `documentation/tutorials/Demo_02_Uncertainty_Analysis/demo_outdir/`). This directory will contain the SIPNET binary as well as the output generated by PEcAn in this demo.
```

Comment:
```suggestion
> Note: The `demo_outdir` directory will be created inside of your PEcAn installation, at `documentation/tutorials/Demo_02_Uncertainty_Analysis/demo_outdir/`. This directory will contain the SIPNET binary as well as the output generated by PEcAn in this demo.
```

(For the record I don't love this outdir placement, but I think we already dicussed that in the PR for demo 1. My chnge here is just to describe the existing placement more clearly)